src/main/java/hyperledger/besu/java/client/ethconnecter/controller/TransactionController.java

```diff
@@ -1,60 +1,28 @@
 package hyperledger.besu.java.client.ethconnecter.controller;
 
-import hyperledger.besu.java.client.ethconnecter.model.Transaction;
 import hyperledger.besu.java.client.ethconnecter.service.TransactionService;
 import java.math.BigInteger;
-import java.util.ArrayList;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.validation.annotation.Validated;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.*;
```

Comment:
[nitpick] Please do not wildcard import. This is to avoid unnecessary conflict in case if the same name is used for class naming across packages. Also, to avoid unnecessary packages from being imported into the package.